.circleci/config.yml

```diff
@@ -683,3 +705,15 @@ workflows:
           path: .
           repo: lotus-dev
           tag: '${CIRCLE_SHA1:0:8}'
+      - publish-packer:
+          requires:
+            - build-all
+            - build-ntwk-calibration
+          filters:
+            branches:
+              ignore:
+                - /.*/
+            tags:
```

Comment:
When I initially wrote this, the filters matched the behavior of the docker image pusher, but the docker publisher was changed since this was written.